dataavailability/datacommittee/datacommittee_test.go

```diff
@@ -0,0 +1,127 @@
+package datacommittee
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygonHermez/zkevm-node/etherman/smartcontracts/polygondatacommittee"
+	"github.com/0xPolygonHermez/zkevm-node/log"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind/backends"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUpdateDataCommitteeEvent(t *testing.T) {
+	// Set up testing environment
+	dac, ethBackend, auth, da := newTestingEnv()
+
+	// Update the committee
+	requiredAmountOfSignatures := big.NewInt(2)
+	URLs := []string{"1", "2", "3"}
+	addrs := []common.Address{
+		common.HexToAddress("0x1"),
+		common.HexToAddress("0x2"),
+		common.HexToAddress("0x3"),
+	}
+	addrsBytes := []byte{}
+	for _, addr := range addrs {
+		addrsBytes = append(addrsBytes, addr.Bytes()...)
+	}
+	_, err := da.SetupCommittee(auth, requiredAmountOfSignatures, URLs, addrsBytes)
+	require.NoError(t, err)
+	ethBackend.Commit()
+
+	// Assert the committee update
+	actualSetup, err := dac.getCurrentDataCommittee()
+	require.NoError(t, err)
+	expectedMembers := []DataCommitteeMember{}
+	expectedSetup := DataCommittee{
+		RequiredSignatures: uint64(len(URLs) - 1),
+		AddressesHash:      crypto.Keccak256Hash(addrsBytes),
+	}
+	for i, url := range URLs {
+		expectedMembers = append(expectedMembers, DataCommitteeMember{
+			URL:  url,
+			Addr: addrs[i],
+		})
+	}
+	expectedSetup.Members = expectedMembers
+	assert.Equal(t, expectedSetup, *actualSetup)
+}
+
+func init() {
+	log.Init(log.Config{
+		Level:   "debug",
+		Outputs: []string{"stderr"},
+	})
+}
+
+// This function prepare the blockchain, the wallet with funds and deploy the smc
+func newTestingEnv() (
+	dac *DataCommitteeBackend,
+	ethBackend *backends.SimulatedBackend,
+	auth *bind.TransactOpts,
+	da *polygondatacommittee.Polygondatacommittee,
+) {
+	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		log.Fatal(err)
+	}
+	auth, err = bind.NewKeyedTransactorWithChainID(privateKey, big.NewInt(1337))
+	if err != nil {
+		log.Fatal(err)
+	}
+	dac, ethBackend, da, err = newSimulatedDacman(auth)
+	if err != nil {
+		log.Fatal(err)
+	}
+	return dac, ethBackend, auth, da
+}
+
+// NewSimulatedEtherman creates an etherman that uses a simulated blockchain. It's important to notice that the ChainID of the auth
+// must be 1337. The address that holds the auth will have an initial balance of 10 ETH
+func newSimulatedDacman(auth *bind.TransactOpts) (
```

Comment:
Pass `testing.T` to it, and set `t.Helper()` at the beginning of the function to mark it as test helper function.